sale_commission_partial_settlement/models/account_partial_reconcile.py

```diff
@@ -5,3 +5,8 @@ class AccountPartialReconcile(models.Model):
     _inherit = "account.partial.reconcile"
 
     partial_commission_settled = fields.Boolean()
+
+    def _skip_future_partial_payments(self, date_payment_to, counterpart_line_date):
```

Comment:
There's not too much sense to put here this method, as `self` is not used at all.